week-3/Homework/mandatory/1-oxygen-levels.js

```diff
@@ -9,32 +9,33 @@ To be safe, they need to land on the first unamed planet that has Oxygen levels
 Write a function that finds the oxygen level of the first safe planet - Oxygen between 19.5% and 23.5%
 */
 
-function safeLevels() {
-
+function safeLevels(planet) {
+  let oxyPlanet = planet.find(x => x > "19.5%" && x < "23.5%");
```

Comment:
You're really close, but I don't think this is quite right.

The problem here is that you're using the greater than and less than operators with strings not numbers.

Try changing one of the values in the test to "100.5%". Does it do what you expected? How could you fix it?

Reply:
Note that this appears to work because `>` does actually work with strings. It just checks **alphabetically** instead of numerically, which is why 100.5% will break it.

Reply:
Hi, yes

> Well done @AlexandruBudaca! 🎆 🌻 🏅
> 
> Have you tried the extra exercise?

I think about it, but I didn't try :( I will

Reply:
Yes we remade the exercise in class. we used parseFloat(), but I didn't remake the exercise.